src/client.ts

```diff
@@ -76,7 +78,128 @@ export default function connect(
   dsn?: string | ConnectConfig | null,
   options?: ConnectConfig | null
 ): Promise<Connection> {
-  return ConnectionImpl.connect(dsn, options);
+  let config: ConnectConfig | null = null;
+  if (typeof dsn === "string") {
+    config = {...options, dsn};
+  } else {
+    if (dsn != null) {
+      // tslint:disable-next-line: no-console
+      console.warn(
+        "`options` as the first argument to `edgedb.connect` is " +
+          "deprecated, use " +
+          "`edgedb.connect('instance_name_or_dsn', options)`"
+      );
+    }
+    config = {...dsn, ...options};
+  }
+  return StandaloneConnection.connect(parseConnectArguments(config));
+}
+
+function sleep(durationMicros: number): Promise<void> {
+  return new Promise((accept, reject) => {
+    setTimeout(() => accept(), durationMicros);
```

Comment:
setTimeout's argument is in milliseconds.

Reply:
But I suggest to not include timeout type in the name -- in JS ecosystem all timeouts by default are milliseconds, so suffixes like Milli or Micro just add noise.

Reply:
Well, `Micros` instead of `Millis` is for some reason my common typo :( For the suffix itself, see slack discussion.

Reply:
> For the suffix itself, see slack discussion.

Yeah, seen that. Need to think a bit before replying.